app/assets/javascripts/discourse/app/widgets/post-cooked.js

```diff
@@ -95,6 +95,26 @@ export default class PostCooked {
       return;
     }
 
+    // find the best a in each onebox and display link counts only for
```

Comment:
I think we need to provide more details here for the comment to be useful for future readers. What does "best a" means here? From the code, it seems like we want the onebox with the most links?

Reply:
It refers to the best <a> element, the one that is the most important to the viewer (for example, heading over paragraph link).